### ISSUE TITLE
How to train my own datasets?

### ISSUE THREAD

**Issue by lily10086** [DeepSegment/ConvNet-Segment#3]
Hello, thank you for sharing the code,But I have a question to ask you.
When I train my own datasets, when I set  tf.flags.DEFINE_string( 'skip_layers) to 32s,26s,8s seperately.
How I set tf.flags.DEFINE_string('trainable_scopes', ...)?
Thanks

**Comment by txzhao** [comment]
Hi, thanks for starring our work! 

We trained our network by a '2-step' procedure, which consists of first freezing all the parameters of low-level layers and only training those of the output layer and upsampling layer on the top, and then fine-tuning all the in-network parameters. In this manner, we first set 'trainable_scopes' to 'InceptionV3/Logits,InceptionV3/Upsampling' in the first stage and then set it to 'None' when fine-tuning all parameters. You can find a specific instruction in the subsection **Fine-tune whole net** of README.md. 

If you have further questions, please don't hesitate to contact us. Cheers!

**Comment by lily10086** [comment]
Thanks,But when I stop to train first step?Or whether I can or not skip first step?

**Comment by txzhao** [comment]
Sorry for such a late reply! 
Yes you can skip the first step but in that case your network won't have a warm start, so the learning process will last longer before convergence. To train the network by a 2-step procedure, we just simply run inception_FCN.py twice with different flag settings as I mentioned above.

**Comment by lily10086** [comment]
Thanks a lot.
